window.go

```diff
@@ -166,8 +166,7 @@ var (
 func initializeWindowPositionIfNeeded(width, height int) {
```

Comment:
Optional: would it be possible to replace `initializeWindowPositionIfNeeded` in `RunGameWithOptions` with `SetMonitor`? If this seems difficult or troublesome to you, I'll care this later.

Reply:
I'll look into it and let you know if I have trouble.

Reply:
I'm not entirely certain how to go about this. `initializeWindowPositionIfNeeded` basically sets the position to the monitor center if an explicit position is not provided. However, `SetMonitor` has no knowledge of if the user has set a position explicitly or not, since that information is provided by `windowPositionSetExplicitly`...

I think `ui.userInterfaceImpl` needs an `initWindowExplicitPosition` field to make this work. Then, in `initOnMainThread(...)`, this position would be set.

It might be quicker/easier for you to do this, as I don't have a full enough view of ebiten's init/run structure yet.

Reply:
Sure, I'll take over this task. Thank you for investigating!